### PR TITLE
[MIG] [11.0] sale_cancel_reason: Backport 12.0 to 11.0

### PR DESCRIPTION
**Pull request by Kiplangatdan**

### CODE REVIEW

File: sale_cancel_reason/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Copyright 2013 Guewen Baconnier, Camptocamp SA
+# Copyright 2019 Victor M.M. Torres, Tecnativa SL
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Sale Cancel Reason',
+    'version': '12.0.1.2.0',
```
Comment (mmequignon) [inline sale_cancel_reason/__manifest__.py]: As you ported this from 12.0 to 11.0, you need to change the module version too
```suggestion
    'version': '11.0.1.0.0',
```
This is why your build is currently failing.

Comment (Kiplangatdan) [reply sale_cancel_reason/__manifest__.py]: @mmequignon Ah yes, of course. Thanks.

**Comment by rousseldenis** [comment]
@Kiplangatdan Could you check pre-commit ?

**Comment by Kiplangatdan** [comment]
@rousseldenis Runbot is green.

**Comment by rousseldenis** [comment]
@Kiplangatdan Could you squash your commits ?

**Review by rousseldenis — APPROVED** [review]

**Comment by rousseldenis** [comment]
@mmequignon Could you update your review?

**Pull request closed without merging**